test/metabase/models/notification_test.clj

```diff
@@ -14,6 +14,18 @@
 
 (use-fixtures :once (fixtures/initialize :web-server))
 
+(def default-system-event-notification
+  {:payload_type :notification/system-event
+   :active       true})
+
+(def default-user-invited-subscription
+  {:type       :notification-subscription/system-event
+   :event_name :event/user-invited})
+
+(def default-card-created-subscription
+  {:type       :notification-subscription/system-event
+   :event_name :event/card-create})
```

Comment:
Have we considered rather having a generic event with the type (e.g. card) as an attribute?

Reply:
yea ideally these tests shouldn't be tied to the registered events, I have `notification.tu/with-temporary-event-topics!` where you can register any event type.

will refactor it in a follow up